Networking/Networking/Remote/Remote.swift

```diff
@@ -93,7 +93,13 @@ public class Remote: NSObject {
             }
 
             guard let data = data else {
-                completion(nil, networkError)
+                let error: Error?  = {
+                    guard let networkError else {
+                        return nil
+                    }
+                    return self.mapNetworkError(error: networkError, for: request)
+                }()
```

Comment:
nit: could it be one line like `let error: Error? = networkError.map { self.mapNetworkError(error: $0, for: request) }`?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/11110/commits/75cb9f5cbaa4db221b989d653ed831d3ebca58da